### PR TITLE
Update phpoffice/phpspreadsheet

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Update phpoffice/phpspreadsheet
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI green and UI tests green
| UI Tests          | https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/11031816016
| Fixed issue or discussion?     | ~
| Related PRs       | ~
| Sponsor company   | ~

### CODE REVIEW

**Review by boherm — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
I think UI tests and CI should be enough to validate this contribution

**Review by tleon — APPROVED** [review]

**Comment by tleon** [comment]
CI & UI tests are green I'm merging this

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
